cmd/summary.go

```diff
@@ -433,6 +434,7 @@ func printSummaryDetails(commitHistory []*model.Commit) error {
 			}
 			if overallStatistics.BreakingModified > 0 {
 				pterm.Info.Printf("Breaking Modifications: %s\n", pterm.LightRed(overallStatistics.BreakingModified))
+				os.Exit(1)
```

Comment:
This is the wrong place to do this. Why for just this breaking type? Why bypass the error return at the bottom? The error us returned to the root command, which performs this exit: https://github.com/pb33f/openapi-changes/blob/main/cmd/root.go#L52